src/main/java/uk/ac/ebi/subs/fileupload/repository/model/File.java

```diff
@@ -32,7 +32,7 @@
     private long uploadedSize;
 
     @Indexed
-    private String user;
+    private String createdBy;
```

Comment:
Why is this field indexed?

Reply:
See below. We might use it in a lookup later.
I can remove it, if you think we don't need it.

Reply:
Removed